REVIEWING.rst

```diff
@@ -0,0 +1,37 @@
+Review Aiven Developer pull requests
+====================================
+
+We have many contributors and reviewers, so here's a checklist to share the things we want to look at when reviewing contributions. Please follow these guidelines when reviewing any pull request on Aiven Developer, and don't merge your own work.
+
+At an overall level
+-------------------
+
+#. Is this a migration from an old knowledgebase article? Make sure to add the old and new URLs to the migrations sheet when merging.
+
+#. If this addresses an open issue, have we added the magic ``Fixes #123`` incantation to the pull request description? (and if it doesn't address an issue, why not?)
+
+#. Does the build pass? We won't merge if it doesn't, but it's also fine for you to either explain to the author how to fix it, or add one commit to fix the link/markup/typo before you merge.
+
+#. Does this change introduce something that our audience actually needs? Not all contributions must be accepted.
+
+#. Do all the articles in the pull request use clear content types, and are they added to the navigation correctly?
+
+#. Are the titles well-written and useful? Are there subtitles?
```

Comment:
Should we have subtitles?

Reply:
Yes, check the guidelines in the "Titles" section of the style guide.